daffodil-test/src/test/resources/org/apache/daffodil/extensions/repType/repType.tdml

```diff
@@ -863,6 +878,11 @@
       <tdml:error>ex:field</tdml:error>
       <tdml:error>facet length (1)</tdml:error>
     </tdml:validationErrors>
+    <tdml:warnings>
+      <tdml:warning>FacetsContradict</tdml:warning>
+      <tdml:warning>'t' contradicts with value of 'minLength' facet</tdml:warning>
+      <tdml:warning>'three' contradicts with value of 'maxLength' facet</tdml:warning>
```

Comment:
I'm ok with this change set. I just find this wording of the warning itself a bit confusing. 

What is contradictory here? Is this trying to issue a warning that 'three' is too long for the maxLength facet? That sounds like a validation error, not a warning, and why doesn't it say that it is too long instead of using this vague 'contradicts with the value of the maxLength facet, but without giving that value of the facet?

Wording wise, "contradicts with value of" should be "contradicts with the value of". I think the article "the" is actually needed in this phrase. But this is one of the reasons I would make this sort of warning check be much less specific. 

I suggest searching for the keywords, not the specific phrasing. Like:
```
<tdml:warnings>
<tdml:warning>FacetContradict</tdml:warning>
<tdml:warning>'t'</tdml:warning>
<tdml:warning>three</tdml:warning>
<tdml:warning>contradicts</tdml:warning>
<tdml:warning>minLength</tdml:warning>
<tdml:warning>maxLength</tdml:warning>
</tdml:warnings>
```

Reply:
I'll update to using the keywords, but I believe this is a xerces error that is then thrown as a warning. So we can't update the actual wording of the warning. I think the issue is that the enums are outside of the bounds of length facets, so it's saying the existing facets contradict each other. As to whether it should be a validationError, I think it makes sense as a warning rather than a validation error, because if you do have contradicting facets that most likely indicates a mistake on the user's side and we will always want to inform them of this. If we made it a validation error, one would need to turn on validation before the possible user error gets caught, which I don't think is right.